docs/source/cloud-setup/cloud-permissions/gcp.rst

```diff
@@ -1,19 +1,56 @@
-.. _cloud-permissions-gcp:
-
 GCP
 =============
 
 
+.. _gcp-service-account:
+
+GCP service account
+-------------------
+
+`GCP service accounts
+<https://cloud.google.com/iam/docs/service-account-overview>`__ are supported.
+
+.. tip::
+    A service account on your local machine can avoid the periodic
+    ``google.auth.exceptions.RefreshError: Reauthentication is needed. Please
+    run `gcloud auth application-default login` to reauthenticate.`` error. A
+    service account is long-lived as it does not have an expiry time.
+
+Set up a service account as follows:
+
+1. Follow the :ref:`instructions <gcp-service-account-creation>` to create a service account with the appropriate roles/permissions.
+2. In the "Service Accounts" tab in the `IAM & Admin console
+   <https://console.cloud.google.com/iam-admin/iam>`__, click on the service
+   account to go to its detailed page. Click on the **KEYS** tab, then click on
+   **ADD KEY** to add a JSON key.  The key will be downloaded automatically.
+3. Set the environment variable ``GOOGLE_APPLICATION_CREDENTIALS`` to the path of the key file, and configure the gcloud CLI tool:
+
+   .. code-block:: console
+
+       $ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json
+       $ gcloud auth activate-service-account --key-file=$GOOGLE_APPLICATION_CREDENTIALS
+       $ gcloud config set project your-project-id
+
+   You may want to add the export statement in your profile (e.g. ``~/.bashrc``, ``~/.zshrc``) so that it is set automatically in all new terminal sessions.
+
+
+
+.. _cloud-permissions-gcp:
+
+Setting permissions
```

Comment:
Since we have service account at the top already, should rename the `Service account` below to `Service account for SkyPilot system`. Also, in AWS page, we may be able to rename the title of IAM role to be `IAM role for SkyPilot system` as well